Providers/nxOMSAutomationWorker/automationworker/3.x/scripts/onboarding3.py

```diff
@@ -189,7 +189,7 @@ def get_autoregistered_worker_account_id():
 
 
 def extract_account_id_from_registration_endpoint(registration_endpoint):
-    account_id = re.findall("[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}",
+    account_id = re.findall("\/accounts\/([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})",
```

Comment:
why is this required now?

Reply:
This is because of the latest private link changes. Now the agent service endpoint has two account ids. The agent service endpoint is account specific. AccountId is prefixed now. So we will have two account ids in the endpoint.